src/utils/get-component-name.ts

```diff
@@ -0,0 +1,13 @@
+// FIXME: Unused module. Verify if it can be freely removed
```

Comment:
Should we remove this util module?

You already found a nicer solution by extracting the name from the compiled output export default function name.

Reply:
yes we can probably remove it.